src/libs/vscode-common/index.ts

```diff
@@ -1,7 +1,8 @@
 export * from "./getExtensionApi";
 export * from "./notebook";
-export * from "./toPlainObject";
+export * from "./testUtil/toPlainObject";
 export * from "./testUtil/testConstants";
+export * from "./vscodeUtil";
```

Comment:
fwiw i lean towards specific exports rather than `*`, but this setup is still pretty new to me so I don't have a strong feeling yet

Reply:
I think I'm on the opposite side. Things that should be public should be in specific files that are exported. And things that should be private should be in its own files. Instead of having to look through files and figure out what's exported and not.

Reply:
That sounds like a reasonable position.  Let's start with that